README.md

```diff
@@ -1,20 +1,187 @@
+    <li>React</li>
+    <li>Redux</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Single Source of Truth**
+- **Virtual DOM**
+- **API to fetch and store SpaceX rockets, Mission**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- To be deployed soon.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need: any web-browser.
+
+### Setup
+
+Clone this [repository](https://github.com/ikoote1/space-travelers-hub) to your desired folder.
+
+### Install
+
+This project does not require installation.
+
+### Usage
+
+**To run the project follow the following instruction:-**
+
+- Clone this [repository](https://github.com/ikoote1/space-travelers-hub) in your local machine with correct branch.
+- First run **npm install**
+- Terminal:- **npm start** it will run on port 8080.
+
+### Run tests
+
+To run tests, open index.html in any web browser.
+
+### Deployment
+
+You can deploy this project using: Project is incomplete.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Ikoote Rasuli**
+
+- GitHub: [@ikoote1](https://github.com/ikoote1)
+
+👤 **Ismail Munyentwari**
+- GitHub: [@ismailmunyentwari9](https://github.com/ismailmunyentwari9)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Support any device resolution]**
+- [ ] **[API compatibility]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give a ⭐️.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **[Can I use this project for my website?]**
+
+  - [It's MIT license, feel free to use as you like.]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] When I click the link to the `MIT.md` file you added, I get this 404 `error`. Please update your Readme, it will be nice to update this section(`License`) with the type of License and a link to the license file, making your readme more informative and professional. Check out this [template](https://github.com/microverseinc/readme-template) for more clarifications.

- [ ] When I click the link to the `MIT.md` file that you added, I get this 404 `error`: